docs/getting-started.md

```diff
@@ -257,29 +251,37 @@ to learn how to connect.
 
 <a class="btn btn-md btn-primary" href="/docs/tutorials/connecting-to-ev3dev-with-ssh" target="_blank"><span class="glyphicon glyphicon-transfer"></span> Learn how to connect with SSH</a>
 
-To test this out, let's try running a command. Type (or copy-and-paste) the
-following into the terminal, and press enter. It will use a nifty program called
-`fortune` to print out a random quote from a database. Feel free to try it
-multiple times!
+To test this out, let's try running a command (make sure that you have completed
+the earlier part of this step first).
 
-{% highlight bash %}
-fortune
+Type the command `fortune` into the SSH shell opened above and press
+<kbd>Enter</kbd>. It will use a nifty program called `fortune` to print out a
+random quote from a database. Feel free to try it multiple times!
+
+{% highlight none %}
```

Comment:
Apparently, `none` is no good. If you really want no highlighting, then just indent 4 spaces and get rid of the highlight liquid tags.